File: JavaScript/string/IsPalindromeNum.js
/**
 * @param num: a positive number
 * @return: true if it's a palindrome or false
 */
const isPalindrome = function (num) {
    let s = num.toString();
    for (let i = 0; i < s.length / 2; i++) {
        if (s[i] != s[s.length - 1 - i]) {
            return false;
        }
    }
    return true;
}

File: Python/string/IsPalindromeNum.py
class Solution:
    """
    @param num: a positive number
    @return: true if it's a palindrome or false
    """
    def isPalindrome(self, num):
        # write your code here
        list1 = []
        while num:
            list1.append(num%10)
            num = int(num/10)
            
        i, j = 0, len(list1)-1
        while i < j:
            if list1[i] != list1[j]:
                return False
            i, j = i+1, j-1
            
        return True
